docs/docs/code/nodejs/browser-automation/README.md

```diff
@@ -0,0 +1,523 @@
+
+```javascript
+import { playwright } from '@pipedream/browsers';
+
+export default defineComponent({
+  async run({ $ }) {
+    const browser = await playwright.browser();
+    const page = await browser.newPage();
+    await page.goto('https://pipedream.com');
+    const content = await page.content();
+
+    await page.context().close();
+    await browser.close();
+
+    return content;
+  },
+});
+```
+
+:::
+
+::::
+
+#### Submit a form
+
+Playwright can also programmatically click and type on a webpage.
+
+:::: tabs :options="{ useUrlFragment: false }"
+
+::: tab "With Page.type"
+```javascript
+import { playwright } from '@pipedream/browsers';
+
+export default defineComponent({
+  async run({ steps, $ }) {
+    const browser = await playwright.browser();
+    const page = await browser.newPage();
+
+    await page.goto('https://example.com/contact');
+    await page.type('input[name=email]', 'pierce@pipedream.com');
+    await page.type('input[name=name]', 'Dylan Pierce');
+    await page.type('textarea[name=message]', "Hello, from a Pipedream workflow.");
+    await page.click('input[type=submit]');
+
+    await page.context().close();
+    await browser.close();
+  },
+});
+```
+:::
+
+::: tab "With Page.click and Page.keyboard.type"
+```javascript
+import { playwright } from '@pipedream/browsers';
+
+export default defineComponent({
+  async run({ steps, $ }) {
+    const browser = await playwright.browser();
+    const page = await browser.newPage();
+
+    await page.goto('https://example.com/contact');
+    await page.click('input[name=email]')
+    await page.keyboard.type('pierce@pipedream.com');
+    await page.click('input[name=name]')
+    await page.keyboard.type('Dylan Pierce');
+    await page.click('textarea[name=message]')
+    await page.keyboard.type("Hello, from a Pipedream workflow.");
+    await page.click('input[type=submit]');
+
+    await page.context().browser();
+    await browser.close();
+  },
+});
+```
+:::
+
+::::
+
+
+
+## Frequently Asked Questions
+
+### Can I use this package in sources or actions?
+
+Yes, the same `@pipedream/browsers` package can be used in [actions](https://pipedream.com/docs/components/quickstart/nodejs/actions/) as well as [sources](https://pipedream.com/docs/components/quickstart/nodejs/sources/).
+
+The steps are the same as usage in Node.js code. Open a browser, create a page, and close the browser at the end of the code step.
+
+:::warning Memory limits
+
+At this time it's not possible to configure the allocated memory to a Source. You may experience a higher rate of Out of Memory errors on Sources that use Puppeteer or Playwright due to the high usage of memory required by Chromium.
+
+:::
+
+### Workflow exited before step finished execution
+
+Remember to close the browser instance _before_ the step finishes. Otherwise, the browser will keep the step "open" and not transfer control to the next step.
+
+### Out of memory errors or slow starts
```

Comment:
Like the Marketplace pages, I would move this into its own dedicated section above since all users should probably expect to increase memory.